### DISCUSSION TITLE
Key Mismatch

### DISCUSSION THREAD

**Discussion by RakanRocky** [rustdesk/rustdesk#10163] [General]
We keep getting a key mismatch error when attempting to connect to most machines. I have re-installed the client and it works temporarily and then the error returns. This issue has happened after updating the software. 
![image](https://github.com/user-attachments/assets/eaa14a02-3676-4482-a824-1d3b13c95e89)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
